cmake/MakeLuaPath.cmake

```diff
@@ -0,0 +1,53 @@
+# tarantool_make_lua_path provides a convenient way to define
+# LUA_PATH and LUA_CPATH variables.
+#
+# Example usage:
+#
+#   tarantool_make_lua_path(LUA_PATH
+#     PATH
```

Comment:
Should be `PATHS`, as I understood from the code.

Reply:
Right, fixed a comment:


```diff
--- a/cmake/MakeLuaPath.cmake
+++ b/cmake/MakeLuaPath.cmake
@@ -4,7 +4,7 @@
 # Example usage:
 #
 #   tarantool_make_lua_path(LUA_PATH
-#     PATH
+#     PATHS
 #       ./?.lua
 #       ${CMAKE_BINARY_DIR}/?.lua
 #       ${CMAKE_CURRENT_SOURCE_DIR}/?.lua
```